libraries/ui-library/src/components/six-rating/six-rating.scss

```
@@ -0,0 +1,33 @@
+@import '../../../src/global/component';
+
+:host {
+  --six-rating-color: var(--six-color-web-rock-500);
+  --six-rating-color-active: var(--six-color-success-600);
```

```suggestion
  --six-rating-color-active: var(--six-color-red);
```
I would propose to change the default color to the six red to align with the general design of all the components.

---

Yes, I would also say default color is red, unless we say it's generally a rating component then the usual color everyhwere is yellowish :)